### DISCUSSION TITLE
Option to remove trend arrow

### DISCUSSION THREAD

**Discussion by esquishesque** [NightscoutFoundation/xDrip#2346] [Ideas]
My partner is relatively new (1 year) to CGMs after many years of managing T1D with regular glucose meters.  They love having the continuous data but are finding the trend arrows very stressful, and often leading to overcompensating.  I think this is because they already know what to do with a certain glucose level, but then the arrow makes them think they need to do more.  And because the CGM lags behind a bit, it's just anxiety-inducing to see for example the arrow going down when they've already eaten and can feel their blood sugar going back up.

**Comment by Navid200** [comment]
xDrip is already an extremely flexible app.
You can customize the trend arrow image including shape and color.  https://navid200.github.io/xDrip/docs/Display/TrendArrow
Please see if you can customize the arrow to make it less distracting.

There are many requests to add more customization options to xDrip.  The more settings we add to xDrip, the harder it becomes to maintain xDrip and to document it.  So, every request should be questioned to evaluate how necessary it is.

Most users either find the arrow useful or can just ignore it.  Can you not customize the shape of the arrow to help you ignore it?

**Comment by esquishesque** [comment]
Yes, we tried to customize the arrow to make it less salient, but it only
affected the arrow in the app, not the arrow that appears in the pinned
notification, which is the one that is actually seen all the time.

I definitely understand it might be a niche concern and I know nothing
about how complicated it is!  I just thought I'd ask.  Appreciate your help.

**Reply by Navid200** [reply]
Would you please post a screenshot of the notification to clarify exactly what you are talking about?
